department-of-veterans-affairs/caseflow
# Queue | Ready for judge review

Resolves #4263

### Description
Adds a start-checkout-flow dropdown to `AttorneyTaskTable`

### Acceptance Criteria
- [x] Verify that there is a column labeled "Action"
- [x] Verify that clicking the drop down expands and shows two options: 
        - "Ready for Decision Review" 
        - "Ready for OMO Review"
- [x] Verify that when "Ready for Decision Review" is selected the user is taken to "Select Dispositions" in the same window #4471 
- [x] Verify that when user select "Ready for OMO Review" it takes them to the "Submit OMO Page" in #4465 

### Testing Plan
1. Go to `/queue` with `:queue_phase_two` FeatureToggle enabled.
2. Select OMO or Draft Decision checkout flow from table, confirm breadcrumbs appear correctly
3. Verify footer back button text reads "< Back to Your Queue".
4. Click back footer button, confirm you are taken back to `/queue`

![checkout_from_table](https://user-images.githubusercontent.com/8533004/39434895-e97da794-4c67-11e8-899d-b97ef0f9f485.gif)
